.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: php
+
 php:
   - 7.0
   - 7.1
+  - 7.2
+
+env:
+  - COMPOSER_FLAGS=--prefer-lowest
+  - COMPOSER_FLAGS=
```

Comment:
Why `COMPOSER_FLAGS` twice?

Reply:
Because we want to test deps with lowest and latest deps. See here https://travis-ci.org/iro88/phpunit-coverage-check/builds/275639192?utm_source=github_status&utm_medium=notification